hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/DateTieredStoreEngine.java

```diff
@@ -94,6 +94,7 @@ public void forceSelect(CompactionRequestImpl request) {
         throws IOException {
       if (request instanceof DateTieredCompactionRequest) {
         return compactor.compact(request, ((DateTieredCompactionRequest) request).getBoundaries(),
+          ((DateTieredCompactionRequest) request).getBoundariesPolicies(),
```

Comment:
Cast it to DateTieredCompactionRequest with a locl variable and then make use of the casted instance?

Reply:
Yeah... I follow the use of `((DateTieredCompactionRequest) request).getBoundaries()` previous line of code.

Reply:
Use a local variable and no need to cast twice...